pages/docs/concepts/asyncapi-document/resuable-parts.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Reusable parts
+weight: 249
+---
+
+Reusable parts in AsyncAPI provide flexibility, modularity, and code reusability. Using reusable parts makes it easier to generate code and validate the specifications. You can reuse specific document sections such as Messages or schema definitions.
+
+Reusable parts allow you to split up the AsyncAPI document into many files and reference them using the Reference Object ($ref). You can use the $ref keyword to reference the same or another local file or external URL.
```

Comment:
In line 8, you wrote `($ref)`, but this seems to be the only place where you added parenthesis. If you review your work on the Netlify preview link, you'll notice that adding parenthesis created a different markup style where it applies a grey highlight over the text.

To me, it seems you need to **remove the parenthesis** in that line in order to be consistent in all mentions.

<img width="400" alt="Screen Shot 2023-08-31 at 5 02 54 PM" src="https://github.com/asyncapi/website/assets/19964402/cc7b43af-d122-48ee-a6ae-45376ea3a000">

Reply:
I added the parenthesis in the first paragraph for our reader to know, Reference Object means $ref